### PR TITLE
fix issue in schema validator

### PR DESCRIPTION
**Pull request by dushansilva**
Fixes https://github.com/wso2/product-apim/issues/10149

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/SchemaValidator.java [hunk 1]
```diff
@@ -75,8 +74,8 @@ public boolean handleRequest(MessageContext messageContext) {
             return true;
         }
         OpenApiInteractionValidator validator = getOpenAPIValidator(swagger);
-        Request request;
-        request = OpenAPIRequest.from(messageContext);
+        OpenAPIRequest request;
+        request = new OpenAPIRequest(messageContext);
```
Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/SchemaValidator.java:78]:
```suggestion
        OpenAPIRequest request = new OpenAPIRequest(messageContext);
```

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/SchemaValidator.java [hunk 2]
```diff
@@ -97,7 +96,7 @@ public boolean handleResponse(MessageContext messageContext) {
         String swagger = messageContext.getProperty("OPEN_API_STRING").toString();
         OpenApiInteractionValidator validator = getOpenAPIValidator(swagger);
         OpenAPIResponse response;
-        response = OpenAPIResponse.from(messageContext);
+        response = new OpenAPIResponse(messageContext);
```
Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/SchemaValidator.java:99]: same here

Comment (dushansilva) [reply components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/SchemaValidator.java]: Fixed https://github.com/wso2/carbon-apimgt/pull/10170/commits/4d6ea1635c04a69accb13fda028a12f0ca8b063d

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/SchemaValidator.java (hunk 1 above)
Comment (dushansilva) [reply components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/SchemaValidator.java]: Fixed https://github.com/wso2/carbon-apimgt/pull/10170/commits/4d6ea1635c04a69accb13fda028a12f0ca8b063d

**Review by tharindu1st — APPROVED** [review]

**Pull request merged**